source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -76,6 +76,17 @@ You could also connect a customer-relationship management (CRM) or case manageme
 
 Read more about [reporting](/reporting/#reporting) and [refunding payments](/refunding_payments/#refunding-payments).
 
+### When you'll receive payments
+
+You'll receive a payment in your bank account within 2 business days of your user completing their payment.
+
+It may take 3 business days if:
+
+- your user completed their payment after 11pm
```

Comment:
not sure if this is by PSP and something we can definitely say...?

Reply:
This was already in the docs (it's in the second paragraph below the screenshot here: https://docs.payments.service.gov.uk/payment_flow/#payment-successful). I think Katie and I added it a while back and we haven't seen any issues since, so is it to leave on that basis?

Reply:
We've amended this to midnight UTC now.